### PR TITLE
Initialize repo

### PR DESCRIPTION
**Pull request by electrocnic**

### CODE REVIEW

File: VulkanLaunchpad.h [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 #include <iostream>
 #include <memory>
+#include <cstring>
```
Comment (johannesugb) [inline VulkanLaunchpad.h:11]: What was `<cstring>` required for? Or is it not required?

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,189 +1,114 @@
...
-# Target name and Output directory
-################################################################################
-# ARCHIVE_OUTPUT_DIRECTORY is used for static libraries on all platforms and import libraries associated with DLLs on Windows.
-
-if(BUILD_STATIC_LIBS)
-  set_target_properties(${PROJECT_NAME}_static PROPERTIES
-    OUTPUT_NAME "${PROJECT_NAME}_static_$<CONFIG>"
-    ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/${Vkl_Library_Output_Directory}"
-  )
+set(CMAKE_CXX_STANDARD 17)
```
Comment (johannesugb) [inline CMakeLists.txt:37]: Does it mean that we require C++ 17? Or does it work with an older standard too?

Comment (fknfilewalker) [reply CMakeLists.txt:37]: yes, never tried it with c++14

**Review by johannesugb — APPROVED** [review]
LGTM

**Pull request merged**